cli/tests/unit/webcrypto_test.ts

```diff
@@ -25,7 +25,7 @@ unitTest(async function testImportArrayBufferKey() {
 unitTest(async function testSignVerify() {
```

Comment:
Let's just duplicate the test. The conditional branching is confusing (especially with the name).